crates/uv-cli/src/lib.rs

```diff
@@ -534,8 +534,19 @@ pub struct VersionArgs {
     pub value: Option<String>,
 
     /// Update the project version using the given semantics
+    ///
+    /// This flag can be passed multiple times, and the bumps will be applied in the
+    /// following order that prevents bumps from being undone by other bumps:
+    ///
+    ///     major > minor > patch > stable > alpha > beta > rc > post > dev
+    ///
+    /// When using the prerelease modes (alpha, beta, rc), usage will typically look like:
+    ///
+    /// 1. ship a beta of the next release: `uv version --bump patch --bump beta`
+    /// 2. ship another beta version: `uv version --bump beta`
+    /// 3. ship the stable version: `uv version --bump stable`
```

Comment:
I would just move this kind of "workflow example" documentation into the actual docs instead of the CLI reference instead. I'd stick to semantics here.